etc/workflows/studio-upload.xml

```diff
@@ -0,0 +1,448 @@
+                    <label for="publishToEngage">Opencast Media Module</label>
+                  </li>
+                  <li>
+                    <input id="publishToAws" name="searchPublishType" type="radio" class="configField" value="true"/>
+                    <label for="publishToAws">AWS</label>
+                  </li>
+                </ul>
+              </fieldset>
+            </li>
+            <li>
+              <input id="publishToApi" name="publishToApi" type="checkbox" checked="checked" class="configField" value="true" />
+              <label for="publishToApi">External Applications</label>
+            </li>
+            <li>
+              <input id="publishToOaiPmh" name="publishToOaiPmh" type="checkbox" class="configField" value="false" />
+              <label for="publishToOaiPmh">OAI-PMH Default Repository</label>
+            </li>
+          </ul>
+        </fieldset>
+      </div>
+
+      <script type="text/javascript">
+        $('input[name="holdOrPublishToggle"]').change(function(){
+            var show = $(this).val() == 'publish';
+            $('#publicationConfigurationPanel').toggle(show);
+            $('#straightToPublishing').val(show);
+            // Trigger update of the schedule's workflow config
+            $('#straightToPublishing').change();
+        });
+
+        <!-- called on loading an exisiting event -->
+        $('#straightToPublishing').bind('updateConfigUI', function() {
+            var publish = $(this).val().toLowerCase() === 'true';
+            $('#publicationConfigurationPanel').toggle(publish);
+            $('#holdOrPublishToggle_' + (publish ? 'publish':'hold')).prop('checked', 'checked');
+        });
+
+        <!-- Remove the line below if you wish to publish to AWS S3 -->
+        $('#publishToAws').prop('disabled', true);
+        <!-- Remove the line above if you wish to publish to AWS S3 -->
+
+        awsWasDisabled = $('#publishToAws').prop('disabled');
+        awsWasChecked = $('#publishToAws').prop('checked');
+        engageWasChecked = $('#publishToEngage').prop('checked');
+
+        $('input[name="publishToSearch"]').change(function(){
+            if ($(this).is(':checked')) {
+              $('#publishToAws').prop('checked', awsWasChecked);
+              $('#publishToEngage').prop('checked', engageWasChecked);
+              $('#publishToAws').prop('disabled', awsWasDisabled);
+              $('#publishToEngage').prop('disabled', false);
+            } else if (!$(this).is(':checked')) {
+              awsWasChecked = $('#publishToAws').prop('checked');
+              engageWasChecked = $('#publishToEngage').prop('checked');
+
+              $('#publishToAws').prop('disabled', true);
+              $('#publishToEngage').prop('disabled', true);
+              $('#publishToAws').prop('checked', false);
+              $('#publishToEngage').prop('checked', false);
+            }
+        });
+      </script>
+    ]]>
+  </configuration_panel>
```

Comment:
This is unnecessary for studio as well

```suggestion
```

Reply:
But neccessarry if we want to use the workflow to proccess studio videos manually upload by the Admin UI user.